libs/damap/src/lib/components/dmp/specify-data/dataset-dialog/dataset-dialog.component.ts

```diff
@@ -1,12 +1,14 @@
 import {Component, Inject} from '@angular/core';
-import {UntypedFormControl, UntypedFormGroup} from '@angular/forms';
+import { UntypedFormControl, UntypedFormGroup} from '@angular/forms';
 import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
 import {FILE_SIZES, FILE_TYPES} from '../data-specs';
 import {FormService} from '../../../../services/form.service';
 import {Dataset} from '../../../../domain/dataset';
 import {DataSource} from '../../../../domain/enum/data-source.enum';
 import {IdentifierType} from '../../../../domain/enum/identifier-type.enum';
-import {Identifier} from '../../../../domain/identifier';
+import { Identifier } from '@damap/core';
```

Comment:
@lpandath pretty sure this line is why. You'll have to use a relative path here as this will create a circular dependency (can't import the lib into itself).